src/libraries/Directory.Build.targets

```diff
@@ -92,6 +92,15 @@
       <RuntimePath>$(TestHostRootPath)</RuntimePath>
     </BinPlaceTargetFrameworks>
 
+    <!-- Setup the runtime pack structure for mobile platforms (for testing and installer) -->
+    <BinPlaceTargetFrameworks Condition="'$(BinPlaceTestRuntimePack)' == 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
+      <RuntimePath>$(NETCoreAppRuntimePackLibPath)</RuntimePath>
+    </BinPlaceTargetFrameworks>
+    <BinPlaceTargetFrameworks Condition="'$(BinPlaceTestRuntimePack)' == 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
+      <RuntimePath>$(NETCoreAppRuntimePackNativePath)</RuntimePath>
+      <ItemName>NativeRuntimePackBinPlaceItem</ItemName>
+    </BinPlaceTargetFrameworks>
```

Comment:
the two `BinPlaceTargetFrameworks` nodes seem identical except for the child nodes, is that intentional?

Reply:
Yes, it's a way to split the bin place copies should they have an <ItemName> of `NativeRuntimePackBinPlaceItem`.  Assemblies go to lib and libMono (and friends) + system.native libraries go to native.

Reply:
I think this deserves a comment explaining what is the difference in between the two.